game/missiongenerator/aircraft/flightgroupspawner.py

```diff
@@ -95,7 +95,7 @@ def generate_flight_at_departure(self) -> FlyingGroup[Any]:
         name = namegen.next_aircraft_name(self.country, self.flight)
         cp = self.flight.departure
         try:
-            if self.start_type is StartType.IN_FLIGHT:
+            if self.start_type is StartType.IN_FLIGHT or isinstance(cp, OffMapSpawn):
```

Comment:
Force the generator to spawn aircraft in the air if they're departing from an "OffMapSpawn" CP

Reply:
This is the bug though. Those should be set to `IN_FLIGHT`.

Please split up the change. I can't submit the other fixes because I can't submit part of a PR.